pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -851,6 +852,22 @@ type PrometheusStatus struct {
 	// +listMapKey=shardID
 	// +optional
 	ShardStatuses []ShardStatus `json:"shardStatuses,omitempty"`
+	// Scale is the Prometheus's scale subresource status.
+	// +optional
+	Scale ScaleSubresourceStatus `json:"scale,omitempty"`
```

Comment:
My preference would be to keep the fields at this level rather than introducing an intermediate struct.